### PR TITLE
Add UDF Db Function tests to EFCore.Relational.Specifications.Tests

### PR DESCRIPTION
**Pull request by pmiddleton**
Add UDF Db Function tests to EFCore.Relational.Specifications.Tests per #12044

I don't have a way to test on Oracle.  Will the automated tests run on Oracle?

### CODE REVIEW

**Comment by tuespetre** [comment]
PAUL

Thank y’all 🤠

File: samples/OracleProvider/test/OracleProvider.FunctionalTests/Query/UdfDbFunctionOracleTests.cs [hunk 1]
```diff
@@ -14,608 +12,409 @@
 
 namespace Microsoft.EntityFrameworkCore.Query
 {
-    public class UdfDbFunctionOracleTests : IClassFixture<UdfDbFunctionOracleTests.OracleUDFFixture>
+    public class UdfDbFunctionOracleTests : UdfDbFunctionTestBase<UdfDbFunctionOracleTests.OracleUDFFixture>
     {
```
Comment (anpete) [inline samples/OracleProvider/test/OracleProvider.FunctionalTests/Query/UdfDbFunctionOracleTests.cs]: Casing of OracleUDFFixture. Lets use "Udf" everywhere

File: samples/OracleProvider/test/OracleProvider.FunctionalTests/Query/UdfDbFunctionOracleTests.cs [hunk 2]
```diff
@@ -14,608 +12,409 @@
...
 
-        private OracleUDFFixture Fixture { get; }
+        #region Scalar Tests
 
-        protected UDFSqlContext CreateContext() => (UDFSqlContext)Fixture.CreateContext();
+        #region Static
 
-        public class Customer
+        [Fact]
+        public override void Scalar_Function_Extension_Method_Static()
```
Comment (anpete) [inline samples/OracleProvider/test/OracleProvider.FunctionalTests/Query/UdfDbFunctionOracleTests.cs]: Remove redundant overrides.

Comment (pmiddleton) [reply samples/OracleProvider/test/OracleProvider.FunctionalTests/Query/UdfDbFunctionOracleTests.cs]: Are there no plans to eventually check the Oracle sql?

Comment (anpete) [reply samples/OracleProvider/test/OracleProvider.FunctionalTests/Query/UdfDbFunctionOracleTests.cs]: Probably not, unless it is different in some way that is worth verifying - we try to only verify it in one place (when we can).

**Comment by pmiddleton** [comment]
updated and rebased.

**Comment by smitpatel** [comment]
Then, it needs to be added to exception list. Tests are failing on it.

**Comment by anpete** [comment]
@pmiddleton Can you add an entry to SqliteComplianceTest.IgnoredTestBases?

**Comment by pmiddleton** [comment]
@anpete - Thank you.  I have been looking for 10 minutes for how that is controlled :)

**Comment by anpete** [comment]
Thanks @pmiddleton!

**Comment by ralmsdeveloper** [comment]
@anpete want me to try to implement these tests for Oracle?

**Comment by anpete** [comment]
@ralmsdeveloper They are there already, right?

**Comment by ralmsdeveloper** [comment]
@anpete
You're right, I confused myself with when @pmiddleton asked.
`Will the automated tests run on Oracle?`

so I figured I'd have some specific test for the provider.

**Comment by pmiddleton** [comment]
@ralmsdeveloper - I wasn't sure if the CI ran the Oracle tests.  I don't have an Oracle instance setup to test on.

**Pull request merged**